templates/cluster-template-ipv6.yaml

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
We should probably remove this template for the time being as we aren't supporting AWSCluster yet